app/controllers/support_interface/candidates_controller.rb

```diff
@@ -63,12 +61,35 @@ def impersonate
       redirect_to candidate_interface_interstitial_path
     end
 
+    def edit_candidate_account_status
+      @candidate = Candidate.find(params[:candidate_id])
```

Comment:
We could put these two new actions behind a feature flag but I don't think it's necessary as this is the support interface.

Reply:
Yes, I don't think is necessary either because we are removing the feature flag in #6382 and leaving it there won't be a problem